index.html

```diff
@@ -94,7 +94,6 @@ <h1>
 var languageCodes = Object.keys(scratchblocks.allLanguages)
 languageCodes.sort()
 languageCodes.forEach(function(code) {
-  if (code === "en") return
```

Comment:
You'll want to make this change in the other repository: https://github.com/scratchblocks/scratchblocks.github.io/

This repo is the library, that one is the homepage. :)

Reply:
Thank you very much.